scipy/special/tests/test_support_alternative_backends.py

```diff
@@ -32,6 +32,13 @@ def test_dispatch_to_unrecognize_library():
 @pytest.mark.parametrize('f_name_n_args', array_special_func_map.items())
 def test_support_alternative_backends(xp, data, f_name_n_args):
     f_name, n_args = f_name_n_args
+
+    if is_jax(xp):
+        if f_name == 'ndtri':
+            pytest.skip("google/jax#20430")
```

Comment:
This one is fixed upstream, I think you can drop the skip?

Reply:
Ah good point. Given the timing I assumed that the fix made it into JAX 0.4.26, but I didn't verify this.

Reply:
I verified that this failure was fixed with JAX 0.4.27, so this skip can be dropped now.